engine/frontend/src/component/CreateEnclaveModal.js

```diff
@@ -11,9 +11,16 @@ export const CreateEnclaveModal = ({enclaveName, handleSubmit, name, setName, ar
   const handleFormSubmit = (e) => {
     e.preventDefault();
     const fetch = async () => {
-      const enclave = await createEnclave(token, apiHost, enclaveName, productionMode);
-      addEnclave(enclave)
-      handleSubmit(enclave);
+      try {
+        const enclave = await createEnclave(token, apiHost, enclaveName, productionMode);
+        addEnclave(enclave)
+        handleSubmit(enclave);
+      } catch(ex) {
```

Comment:
`nit`: is `ex` supposed to stand for exception? used to seeing `err` here, but all the same

Reply:
yeah in javascript; it's in an exception :)